azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/viewmodel/MessageViewModel.kt

```diff
@@ -55,7 +59,8 @@ private class InfoModelToViewModelAdapter(
                 thisMessage.createdOn ?: OffsetDateTime.now()
             ),
 
-            isLocalUser = isLocalUser
+            isLocalUser = isLocalUser,
+            isRead = isLocalUser && (currentMessageTime != null && currentMessageTime <= latestReadMessageTimestamp)
```

Comment:
if isLocalUser is true, doesn't it mean that this message is sent by myself? 
In that case, wouldn't isRead be always true? 
So in this case, if isLocalUser is true and currentMessageTime is greater than latestReadMessageTimestamp, isRead should be true too. 
But from the logic written here, it will be false when it should be true.

Reply:
my bad. this is receiving condition and it looks fine.